app/views/spaces/show.html.erb

```diff
@@ -13,6 +13,9 @@
   <%= render partial: 'spaces/show/basics' %>
   <hr />
 
+  <%= render 'space_contacts/index' %>
```

Comment:
Kult å bare rendre hele indexen til space contacts her! Kanskje vi burde gjøre noe tilsvarende for mange av de andre?

Reply:
Ja, det kan være en god ide. Er lett lesbart.
Jeg liker personlig den måten, men det er mange gode måter å gjøre dette på.